easybuild/easyconfigs/o/OpenBLAS/OpenBLAS-0.2.19-GCC-6.4.0-2.28-LAPACK-3.7.0.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'ConfigureMake'
+
+name = 'OpenBLAS'
+version = '0.2.19'
```

Comment:
Since the 0.2.21 release isn't happening, I went forward with using 0.2.20 and applying two patches on top of it, so we can finalise `foss/2017b`...